modules/commons/src/main/frontend/src/SearchBar.jsx

```diff
@@ -195,22 +193,20 @@ function SearchBar(props) {
         }}
         endAdornment={
           <InputAdornment position="end">
-            <IconButton
-              size="large"
-              className={invertColors ? classes.invertedColors : ""}
-              onClick={(event) => {
-                input?.current?.focus();
-              }}
-            >
-              <Search />
-            </IconButton>
+            {disableButton
+              ? <Search/>
+              : <IconButton
+                  size="small"
+                  sx={{mr: 0.5}}
```

Comment:
The button is getting shifted in the wrong direction, creating an even bigger gap on the right.
```suggestion
                  sx={{mr: -0.5}}
```